packages/api-reference/src/features/InfoObject/TermsOfService.vue

```diff
@@ -0,0 +1,18 @@
+<script setup lang="ts">
+import { type OpenAPIV3_1 } from '@scalar/openapi-types'
+
+defineProps<{
+  value?: OpenAPIV3_1.InfoObject['termsOfService']
+  description?: string
+}>()
+</script>
+
+<template>
+  <template v-if="value">
+    <a
+      :href="value"
+      target="_blank">
+      {{ description || 'Terms of Service' }}
```

Comment:
`termsOfService` is just an URL, we won’t have a description for it, or am I missing something?